test/cypress/fixtures/unique_cmd_parent/main.py

```diff
@@ -0,0 +1,61 @@
+from typing import Any, Type
```

Comment:
I don't like that these are copy-pasted. 

Instead, you can copy these from the following image, and put them in the right place.
```
COPY --from=grapl/etc-build /home/grapl/etc 
```